src/app/views/onboarding/step2.coffee

```diff
@@ -18,8 +18,24 @@ angular.module 'mnoEnterpriseAngular'
 
     # Select or deselect an app
     vm.toggleApp = (app) ->
-      # User cannot add more apps
-      return if vm.maxAppsSelected && !app.checked
+      # Add conflictingApp attribute to marketplace apps
+      app.subcategoryNames = []
+      _.each(app.subcategories, (appSubcategory) ->
+        app.subcategoryNames.push(appSubcategory.name)
+      )
+      app.marketplaceApps = _.filter(vm.marketplace.apps, (marketplaceApp) ->
+        marketplaceApp != app
+      )
+      _.each(app.marketplaceApps, (marketplaceApp) ->
+        _.each(marketplaceApp.subcategories, (marketplaceAppSubCategory) ->
+          _.each(app.subcategoryNames, (subcategoryName) ->
+            if !marketplaceAppSubCategory.multi_instantiable && (marketplaceAppSubCategory.name == subcategoryName)
+              marketplaceApp.conflictingApp = app
```

Comment:
This section is a bit complicated and may be simplified.
Inspire from those examples:
- https://github.com/maestrano/mno-enterprise-angular/blob/1.1/src/app/components/mno-app-onboarding-btn/mno-app-onboarding-btn.coffee
- https://github.com/maestrano/mno-enterprise-angular/blob/1.1/src/app/components/mno-app-install-btn/mno-app-install-btn.coffee